code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -762,6 +762,12 @@
 	if(SM.sentience_type != animal_type)
 		to_chat(user, "<span class='warning'>You cannot transfer your consciousness to [SM].</span>" )
 		return ..()
+	if(SM.mob_biotypes & MOB_ROBOTIC)
```

Comment:
this keeps you from mind transfer potion'ing hivebot engineers, which are in the gold slime core spawn pool and are specifically designed to be player-controlled